internal/extsvc/github/v3.go

```diff
@@ -52,12 +53,15 @@ type V3Client struct {
 	// httpClient is the HTTP client used to make requests to the GitHub API.
 	httpClient httpcli.Doer
 
-	// rateLimitMonitor is the API rate limit monitor.
-	rateLimitMonitor *ratelimit.Monitor
+	// externalRateLimiter is the external API rate limit monitor.
+	externalRateLimiter *ratelimit.Monitor
 
 	// rateLimit is our self-imposed rate limiter
 	rateLimit *ratelimit.InstrumentedLimiter
 
+	// If true, the client will wait and retry a request if external rate limits are encountered
```

Comment:
```suggestion
	// WaitForRateLimit determines whether the client will wait and retry a request if external rate limits are encountered.
```

Nitpick, but this is how you do docstrings in Go, ideally.

Also: does this need to be exported, since you have the setter?